docs/features/network-security-controls/admin-network-policy.md

```diff
@@ -1,303 +1,1325 @@
+ensure it was correctly applied you can do `kubectl describe anp cluster-control`:
+
+```shell
+Name:         cluster-control
+Namespace:    
+Labels:       <none>
+Annotations:  k8s.ovn.org/acl-logging: { "deny": "alert", "allow": "alert", "pass" : "warning" }
+API Version:  policy.networking.k8s.io/v1alpha1
+Kind:         AdminNetworkPolicy
+```
+
+In order to view what's happening with the traffic that matches
+this policy, you can do:
+
+```shell
+kubectl logs -n ovn-kubernetes ovnkube-node-ggrxt ovn-controller | grep acl
+```
+
+and it will show you all the ACL logs logged by ovn-controller container.
+Sample logging output:
+
+```shell
+2024-06-09T19:00:10.441Z|00162|acl_log(ovn_pinctrl0)|INFO|name="ANP:cluster-control:Egress:1", verdict=allow, severity=alert, direction=from-lport: tcp,vlan_tci=0x0000,dl_src=0a:58:0a:f4:02:06,dl_dst=0a:58:0a:f4:02:01,nw_src=10.244.2.6,nw_dst=172.18.0.3,nw_tos=0,nw_ecn=0,nw_ttl=64,nw_frag=no,tp_src=33690,tp_dst=6443,tcp_flags=ack
+2024-06-09T19:00:10.696Z|00163|acl_log(ovn_pinctrl0)|INFO|name="ANP:cluster-control:Egress:1", verdict=allow, severity=alert, direction=from-lport: tcp,vlan_tci=0x0000,dl_src=0a:58:0a:f4:02:03,dl_dst=0a:58:0a:f4:02:01,nw_src=10.244.2.3,nw_dst=172.18.0.3,nw_tos=0,nw_ecn=0,nw_ttl=64,nw_frag=no,tp_src=57658,tp_dst=6443,tcp_flags=psh|ack
+2024-06-09T19:00:10.698Z|00164|acl_log(ovn_pinctrl0)|INFO|name="ANP:cluster-control:Egress:1", verdict=allow, severity=alert, direction=from-lport: tcp,vlan_tci=0x0000,dl_src=0a:58:0a:f4:02:03,dl_dst=0a:58:0a:f4:02:01,nw_src=10.244.2.3,nw_dst=172.18.0.3,nw_tos=0,nw_ecn=0,nw_ttl=64,nw_frag=no,tp_src=57658,tp_dst=6443,tcp_flags=ack
+2024-06-09T19:00:11.386Z|00165|acl_log(ovn_pinctrl0)|INFO|name="ANP:cluster-control:Egress:5", verdict=drop, severity=alert, direction=from-lport: icmp,vlan_tci=0x0000,dl_src=0a:58:0a:f4:02:07,dl_dst=0a:58:0a:f4:02:08,nw_src=10.244.2.7,nw_dst=10.244.2.8,nw_tos=0,nw_ecn=0,nw_ttl=64,nw_frag=no,icmp_type=8,icmp_code=0
+```
+
+The same is applicable to debug baseline admin network policy as well
+except for the fact that there is no `pass` in BANP:
+
+```shell
+kubectl annotate banp default k8s.ovn.org/acl-logging='{ "deny": "alert", "allow": "alert" }'
+```
+
+### Ensuring NBDB objects are correctly created
+
+See the details outlined in the OVN constructs section on
+what are the OVN database constructs created for ANPs and BANP
+in a cluster. For example, in order to troubleshoot the
+`cluster-control` ANP at priority 34, we can list the address-sets,
+acls and port-group like shown in the above section. But
+that will list all the acls, address-sets and port-groups
+in your cluster also used by other features. To narrow it
+down to the specific policy in question you can grep for
+the name of the ANP you care about:
+
+* `ovn-nbctl list acl | grep cluster-control` (Do grep with -A, -B, -C to see the full ACL)
+* `ovn-nbctl list address-set | grep anp-name`
+* `ovn-nbctl list port-group | grep anp-name`
```

Comment:
What about using `ovn-nbctl find` on the `external_ids` like this?
```
ovn-nbctl find ACL 'external_ids{>=}{"k8s.ovn.org/owner-type"=AdminNetworkPolicy,"k8s.ovn.org/name"=cluster-control}'

ovn-nbctl find Address_Set 'external_ids{>=}{"k8s.ovn.org/owner-type"=AdminNetworkPolicy,"k8s.ovn.org/name"=cluster-control}'

ovn-nbctl find Port_Group 'external_ids{>=}{"k8s.ovn.org/owner-type"=AdminNetworkPolicy,"k8s.ovn.org/name"=cluster-control}'
```

Even if the `ovn-nbctl list` is preferred, it would be good that if the ANP is called `cluster-control`, to use that name across the 3 commands.

Reply:
yeah this works too, let me add this bit in.

Reply:
done!